src/sbvr-api/sbvr-utils.coffee

```diff
@@ -688,12 +695,11 @@ exports.handleODataRequest = handleODataRequest = (req, res, next) ->
 		mapSeries body, (bodypart) ->
 			uriParser.parseOData(bodypart)
 			.then controlFlow.liftP (request) ->
-				# Get the full hooks list now that we can. We clear the hooks on
-				# the global req to avoid duplication
-				req.hooks = {}
+				req = cloneReq(req, request)
```

Comment:
Since we overwrite `req` each time it means that any modifications made to it by previous requests in the batch will be carried forward